bundle/metadata/dependencies.yaml

```diff
@@ -2,4 +2,4 @@ dependencies:
   - type: olm.package
     value:
       packageName: openshift-pipelines-operator-rh
-      version: "1.5.2"
+      version: "1.7.1"
```

Comment:
We need to test this version with our operator as a bundle/catalog in a net new cluster, to make sure the issues in 1.6.x versions were actually fixed.

Reply:
I ran some local tests on a brand-new SNO cluster. I was able to do a pipeline run (the operator I passed itself failed the validation) the pipeline run completed.

Reply:
I was able to test this as well, and OCO installed properly, as did `operator-pipelines`. I poked the cluster some, `events` and `logs` from both `OLM` operators and didn't see any errors/warns, so I think they fixed all the `1.6.x` issues for the things we care about and need.